sound/soc/intel/boards/sof_sdw.c

```diff
@@ -199,6 +208,12 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.dai_name = "rt715-aif2",
 		.init = sof_sdw_rt715_init,
 	},
+	{
+		.id = 0x8373,
+		.direction = {true, true},
```

Comment:
Should it be ``.direction = {true, false},`` as max98373 doesn't support capture

Reply:
@bardliao ,  it supports capture. please check codec driver
````
		.capture = {
			.stream_name = "HiFi Capture",
			.channels_min = 1,
			.channels_max = 2,
			.rates = MAX98373_RATES,
			.formats = MAX98373_FORMATS,
		},
````

Reply:
> @bardliao , it supports capture. please check codec driver

Thanks @RanderWang I didn't notice that

Reply:
yes capture is max98373-sdw for speaker feeddback